test/server/cards/01_SOR/units/97thLegionKeepingThePeaceOnSullust.spec.ts

```diff
@@ -0,0 +1,28 @@
+describe('97th Legion Keeping the Peace on Sullust', function() {
+    integration(function(contextRef) {
+        describe('97th Legion\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['97th-legion#keeping-the-peace-on-sullust'],
+                    },
+                    resources: 7,
+                });
+            });
+
+            it('should give +1/+1 for each resource of the controller', function () {
```

Comment:
can you add a test when your resources change during the turn (with resupply or superlaser-technician for example)

Reply:
@VJubert I'll update this with more scenarios, this is just a draft pr so Veld can help me debug the dynamic stats modifier that is not working :)